features/step_definitions/networking.rb

```diff
@@ -1157,17 +1157,21 @@
   end
 end
 
-Given /^I check load-sctp-module in all nodes$/ do 
+Given /^I check load-sctp-module in #{NUMBER} workers$/ do | workers_num |
```

Comment:
is there a use case where we wouldn't want to check all the workers in `cb.workers`?   Can we just loop over the `cb.workers` array?

Reply:
We better have the step either:
```
I check load-sctp-module in all workers
```
or
```
I check load-sctp-module in nodes from the #{SYM} clipboard
```

If we just specify number, then there is no sane way for the scenario to specify which exactly workers we care about. Thus making application very limited. While providing a clipboard with nodes, would allow you to choose what you want.

And if such flexibility is not needed, then just go with the `all workers` one.

Reply:
> is there a use case where we wouldn't want to check all the workers in `cb.workers`? Can we just loop over the `cb.workers` array?

@rbbratta we can loop over 'cb.nodes' , below code works:
env.nodes.each do |node|
    @result = node.host.exec_admin("lsmod \| grep sctp")

But loop over 'cb.workers' not support, below code will not work
env.workers.each do |workers|
    @result = workers.host.exec_admin("cat /sys/module/sctp/initstate")

For sctp testing, we need check sctp module in all worker nodes which not include master nodes, so old code env.nodes.each do |node| will cause test fail.

Reply:
@rbbratta Just got this works:
cb.workers.each do |workers|

Reply:
> We better have the step either:
> 
> ```
> I check load-sctp-module in all workers
> ```
> 
> or
> 
> ```
> I check load-sctp-module in nodes from the #{SYM} clipboard
> ```
> 
> If we just specify number, then there is no sane way for the scenario to specify which exactly workers we care about. Thus making application very limited. While providing a clipboard with nodes, would allow you to choose what you want.
> 
> And if such flexibility is not needed, then just go with the `all workers` one.

Fixed by using "Given I check load-sctp-module in all workers"